src/main/java/uk/gov/hmcts/reform/sendletter/notify/INotifyClient.java

```diff
@@ -0,0 +1,7 @@
+package uk.gov.hmcts.reform.sendletter.notify;
+
+import uk.gov.hmcts.reform.sendletter.model.Letter;
+
+public interface INotifyClient {
```

Comment:
I am not sure if we are following a pattern to name interface prefixed with `I`.If not we can just have the interface name as `NotifiyClient`